src/Microsoft.NET.Sdk.Functions.MSBuild/Targets/Microsoft.NET.Sdk.Functions.Version.props

```diff
@@ -11,7 +11,7 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <FunctionsSdkVersion>1.0.30-beta4</FunctionsSdkVersion>
+    <FunctionsSdkVersion>1.0.30-beta5</FunctionsSdkVersion>
```

Comment:
I'm wondering if this should just be a non-beta release. We have a series of fixes that have been sitting in beta for a while now

Reply:
Yeah I don't actually know why we went the beta route. I think you changed it a little while ago. I'm cool with going 1.0.30 here.

Reply:
Yes, we had some beta releases because of some significant changes that needed some additional validation. But we should be good with a 1.0.30 release here.